scalableminds/webknossos
# Add Infer Mitochondria Job

Adds infer_mitochondria worker job (for now for superusers only)

Corresponding PR in voxelytics: https://github.com/scalableminds/voxelytics/pull/3504

### Steps to test:
- Set up worker with vx branch `infer-mitochondria-workflow`, in wk, run `yarn enable-jobs`, open dataset/annotation in wk, create a bbox with a mito, hit “AI Analysis”, run mito inferral job, result should look ok

------
- [x] (no changelog for now, since this is still behind the superUser condition)
